### ISSUE TITLE
pytorch training issue: insufficient shared memory

### ISSUE THREAD

**Issue by Eric-Zhang1990** [IBM/FfDL#167]
@Tomcli @sboagibm  I want to run maskrcnn-benchmark project (https://github.com/facebookresearch/maskrcnn-benchmark),
![image](https://user-images.githubusercontent.com/24602893/54346834-b81a4b80-4680-11e9-8289-2ef5f0742fa3.png)
--------------------
![深度截图_选择区域_20190314174440](https://user-images.githubusercontent.com/24602893/54346944-e8fa8080-4680-11e9-95a1-e1ab7bb38868.png)
but I find NOTE at https://github.com/IBM/FfDL/blob/master/docs/user-guide.md#24-creating-manifest-file (Note that all model definition files has to be in the first level of the zip file and there are no nested directories in the zip file.), which means I can't run job which has multi directories, right?
However, all our projects have multi directories, how can I run them on FfDL?
Thank you.

**Comment by Tomcli** [comment]
Hi @Eric-Zhang1990, technically you can package your code with the bash `zip` command if you have multi-directories. (e.g. `zip -r model.zip maskrcnn-benchmark`) 

In the backend, FfDL will unzip the model.zip to the learner container's work_dir. Then, you can modify the command in manifest.yml to execute your workload. 
(e.g. `python maskrcnn_benchmark/tools/train_net.py`)

**Comment by Eric-Zhang1990** [comment]
@Tomcli  Thank you for your kind reply. I loss a parameter '-r' when zip model files. 
However, I encounter another problem about 'shm', which is "ERROR: Unexpected bus error encountered in worker. This might be caused by insufficient shared memory (shm).".
my config is:
gpus: 1
cpus: 4
learners: 1
memory: 16Gb

framework:
  name: custom
  version: "pytorch:1.0-gpu"
  command: >
    export NGPUS=1; source /etc/profile; bash ${MODEL_DIR}/setup.sh; . env_file; python ${MODEL_DIR}/setup.py build develop; python -m torch.distributed.launch --nproc_per_node=$NUM_GPUS ${MODEL_DIR}/train_net.py;

Thank you.
